src/Compilers/Core/Portable/CommandLine/ReportAnalyzerUtil.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.CodeAnalysis
+{
+    internal static class ReportAnalyzerUtil
+    {
+        public static void Report(
+            TextWriter consoleOutput,
+            AnalyzerDriver? analyzerDriver,
+            GeneratorDriverTimingInfo? driverTimingInfo,
+            CultureInfo culture,
+            bool isConcurrentBuild)
+        {
+            if (isConcurrentBuild && (analyzerDriver is { } || driverTimingInfo is { }))
+            {
+                consoleOutput.WriteLine(CodeAnalysisResources.MultithreadedAnalyzerExecutionNote);
+                consoleOutput.WriteLine();
+            }
+
+            if (analyzerDriver is object)
```

Comment:
💭 Seems like this is the odd man out

```suggestion
            if (analyzerDriver is { })
```

Would also be fine to change the three `is { }`/`is object` cases to `is not null`.

Reply:
I remain confused why we prefer `is { }` to `is object` in our code base given that it's demonstrably inferior. It allows for us to insert silent `true` into our code base. 

```c#
int i = 42;
Console.WriteLine(i is {});     // silently converted to true
Console.WriteLine(i is object); // Warn
```

What is the rationale behind preferring the pattern which allows more bugs into the codebase?

Reply:
We generally prefer `is not null`. The `is { }` form is typically used instead of `is var` in cases where we want both type inference and a non-null check.